pkg/server/xgrpc/server.go

```diff
@@ -42,6 +46,32 @@ func newServer(config *Config) (*Server, error) {
 		config.unaryInterceptors...,
 	)
 
+	if config.EnableTLS {
+		cert, err := tls.LoadX509KeyPair(config.CertFile, config.PrivateFile)
+		if err != nil {
+			panic(err)
```

Comment:
seem better to use xlog.Panicf instead, or return err directly.

Reply:
fixed